dash/include/dash/matrix/MatrixRef.h

```diff
@@ -9,7 +9,9 @@
 #include <dash/Pattern.h>
 #include <dash/GlobRef.h>
 #include <dash/HView.h>
-#include <dash/Container.h>
+
+#include <dash/view/Local.h>
```

Comment:
Are these required? The implementation of `dash::Matrix` does not depend on view expressions, does it?